verification/curator-service/ui/cypress/integration/components/BulkCaseForm.spec.ts

```diff
@@ -104,8 +109,10 @@ describe('Bulk upload form', function () {
         cy.contains('Success!');
         cy.get('button[aria-label="close overlay"').click();
 
-        cy.visit('/cases');
-        cy.get('tr').should('have.length', 3);
+        // TODO: Reimplement this check, after we remove phantom rows from
+        // the linelist table.
+        // cy.visit('/cases');
+        // cy.get('tr').should('have.length', 3);
```

Comment:
I'm surprised this didn't fail integration tests in the CI for my PR... Is there another check you could do in the mean time? maybe a field all three rows have that you could check there are only 3 instances of

Reply:
Likewise, and good suggestion -- I'll assert on the first column.